src/agent/rustjail/Cargo.toml

```diff
@@ -23,11 +23,11 @@ scan_fmt = "0.2.6"
 regex = "1.5.4"
 path-absolutize = "1.2.0"
 anyhow = "1.0.32"
-cgroups = { package = "cgroups-rs", version = "0.2.5" }
+cgroups = { package = "cgroups-rs", version = "0.2.8" }
 rlimit = "0.5.3"
 
 tokio = { version = "1.2.0", features = ["sync", "io-util", "process", "time", "macros"] }
-futures = "0.3.18"
+futures = "0.3.17"
```

Comment:
From 0.3.18 to 0.3.17?

Reply:
Unfortunately yes - that version was removed - see https://crates.io/crates/futures/versions.